### PR TITLE
Fix(web-react): Tabs styleProps #DS-977

### PR DESCRIPTION
**Pull request by pavelklibani**
## Description

- Added `styleProps` to **TabsContent**, **TabItem**, **TabList** and **TabPane**
- `UNSAFE_className` and `UNSAFE_style` added to API tables for **TabsContent**, **TabItem**, **TabList** and **TabPane**

### Issue reference

[React: Tabs nemají styleProps](https://jira.lmc.cz/browse/DS-977)

### CODE REVIEW

**Review by literat — COMMENTED** [review]
Please, add also tests for the styleProps :-)

**Comment by pavelklibani** [comment]
> Please, add also tests for the styleProps :-)

Tests added

File: packages/web-react/src/components/Tabs/__tests__/TabItem.test.tsx [hunk 1]
```diff
@@ -2,11 +2,16 @@ import '@testing-library/jest-dom';
 import { fireEvent, render, waitFor } from '@testing-library/react';
 import React from 'react';
 import { classNamePrefixProviderTest } from '../../../../tests/providerTests/classNamePrefixProviderTest';
+import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
 import { restPropsTest } from '../../../../tests/providerTests/restPropsTest';
 import { withTabsContext } from '../../../../tests/testUtils/withTabsContext';
+// import Tabs from '../Tabs';
```
Comment (literat) [inline packages/web-react/src/components/Tabs/__tests__/TabItem.test.tsx:8]: Unused code.

File: packages/web-react/src/components/Tabs/__tests__/TabList.test.tsx [hunk 2]
```diff
@@ -1,7 +1,12 @@
 import '@testing-library/jest-dom';
+import React from 'react';
 import { classNamePrefixProviderTest } from '../../../../tests/providerTests/classNamePrefixProviderTest';
+import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
+// import Tabs from '../Tabs';
```
Comment (literat) [inline packages/web-react/src/components/Tabs/__tests__/TabList.test.tsx:5]: Unused code.

File: packages/web-react/src/components/Tabs/__tests__/TabItem.test.tsx (hunk 1 above)
Comment (pavelklibani) [reply packages/web-react/src/components/Tabs/__tests__/TabItem.test.tsx]: removed

File: packages/web-react/src/components/Tabs/__tests__/TabList.test.tsx (hunk 2 above)
Comment (pavelklibani) [reply packages/web-react/src/components/Tabs/__tests__/TabList.test.tsx]: removed

**Pull request merged**